contracts/near/eth2-client/src/tests/unit_tests.rs

```diff
@@ -533,4 +533,37 @@ mod tests {
         update.sync_committee_update = None;
         contract.submit_beacon_chain_light_client_update(update);
     }
+
+    #[test]
+    #[should_panic(
+        expected = "The client can't be executed in (trustless mode) and (without bls) on mainnet."
+    )]
+    pub fn test_panic_on_init_in_trusted_mode_without_bls_on_mainnet() {
```

Comment:
I belive it's a typo
```suggestion
    )]
    pub fn test_panic_on_init_in_trustless_mode_without_bls_on_mainnet() {
```